ui/imports/shared/views/chat/ChatContextMenuView.qml

```diff
@@ -125,26 +134,13 @@ StatusMenu {
         }
     }
 
-    
-    StatusMenu {
```

Comment:
I'm sure this has to stay, @igor-sirotin @jrainville

Reply:
Indeed, why did this go away 😢 
It's under `showDebugOptions`, so won't bother most users.